### PR TITLE
add recipe for r-lda

### PR DESCRIPTION
**Pull request by Stevierene82**
r-lda recipe created using [conda r skeleton helper](https://github.com/bgruening/conda_r_skeleton_helper)

### CODE REVIEW

**Comment by Stevierene82** [comment]
Hi! Looks like the error I'm getting has to do with gcc. Does anyone have any insight/advice in how to go about fixing this? Thanks!

image

File: recipes/r-lda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}        # [not win]
```
Comment (scopatz) [inline recipes/r-lda/meta.yaml]: You want `compiler('c')` in all cases, and don't want to exclude it from windows.  compiler('c') provides all the platform dependent compilers

File: recipes/r-lda/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - {{native}}toolchain        # [win]
```
Comment (scopatz) [inline recipes/r-lda/meta.yaml]: You don't need tohave toolchain if you have compiler('c')

File: recipes/r-lda/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    - {{posix}}filesystem        # [win]
+    - {{posix}}make
+
+  host:
+    - r-base
+
+  run:
+    - r-base
+    - {{native}}gcc-libs         # [win]
```
Comment (scopatz) [inline recipes/r-lda/meta.yaml]: You also don't need this with the proper compiler('c') line

**Comment by Stevierene82** [comment]
Alright, removed the # [not win], as well as the toolchain and gcc-libs lines, but looks like i'm still getting the error for missing gcc file. Did I miss something else? Thank you so much for the help @scopatz!

**Comment by Stevierene82** [comment]
@bgruening - any insight into why the windows build can't find this gcc file? I'm getting this same problem with a couple other packages I've attempted to build, and having trouble figuring this out. Thanks!

**Comment by bgruening** [comment]
Can you run this script again, we fixed recently an error in the windows builds: https://github.com/bgruening/conda_r_skeleton_helper, more concrete it seems we need to have the merge_build_host attribute.

**Comment by Stevierene82** [comment]
@bgruening Thanks for pointing out the updated skeleton! The windows build is now happy - but the linter is not. Any thoughts on that? Thanks!

**Comment by bgruening** [comment]
Not really, besides that we need to fix the linter :(

File: recipes/r-lda/meta.yaml (hunk 2 above)
Comment (bgruening) [reply recipes/r-lda/meta.yaml]: @scopatz are you sure about this? On Windows we need to treat this differently as we need, sed and coreutils and so on.

Comment (scopatz) [reply recipes/r-lda/meta.yaml]: Currently compiler('c') is just a wrapper around toolchain. I think you can install coreutils separately.

Comment (isuruf) [reply recipes/r-lda/meta.yaml]: This expands to `m2w64-toolchain` which is separate from `toolchain`.

Comment (scopatz) [reply recipes/r-lda/meta.yaml]: Ahh true, but don't they still need compiler('c') on win

Comment (isuruf) [reply recipes/r-lda/meta.yaml]: No. `compiler('c')` is for msvc, but r packages use m2w64-gcc

Comment (scopatz) [reply recipes/r-lda/meta.yaml]: Ok, got it.

**Comment by CJ-Wright** [comment]
The linter get the expected subsections [from `conda_build.metadata` as `FIELDS`. ](https://github.com/conda/conda-build/blame/master/conda_build/metadata.py#L491)

I'm not sure why the linter is missing this since it was released recently.

**Comment by jakirkham** [comment]
The linter may need a rebuild to get that update.

**Comment by isuruf** [comment]
@conda-forge-linter please lint

**Comment by jakirkham** [comment]
Thanks @isuruf :)

**Comment by Stevierene82** [comment]
@conda-forge/staged-recipes Requesting review please! Thank you for all the help getting this fixed!

**Pull request merged**